topology/University of Wisconsin–Madison/CHTC/CHTC.yaml

```
@@ -962,4 +962,28 @@ Resources:
     AllowedVOs:
       - OSG
 
+  CHTC-learn-ap:
+    Active: true
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          ID: OSG1000015
+          Name: Aaron Moate
+      Security Contact:
+        Primary:
+          ID: OSG1000015
+          Name: Aaron Moate
+    Description: CHTC Access Point
+    FQDN: learn-ap2000.chtc.wisc.edu
```

```suggestion
    FQDN: learn.chtc.wisc.edu
    FQDNAliases:
    - learn-ap2000.chtc.wisc.edu    
```

Let's keep the same CNAME unless Christina requests otherwise